app-center-webapps/src/main/webapp/skin/less/app-center.less

```diff
@@ -485,230 +485,229 @@
+              padding: 0;
+              margin-bottom: 6px;
+              .v-list-item__content {
+                overflow: unset;
+                padding: 0;
+                height: 45px;
+              }
+              h5 {
+                font-size: 15px;
+                padding-left: 10px;
+              }
+              .image {
+                width: 45px;
+                height: 45px;
+              }
+              .appTitle {
+                max-width: 100px;
+                white-space: nowrap;
+                overflow:hidden;
+                text-overflow:ellipsis;
+                color: @textColorDefault;
+              }
+              .appImage {
+                width: 45px;
+                height: 45px;
+                object-fit: cover;
+                overflow: hidden;
+              }
             }
-            &.favorite {
-              color: @baseBackgroundDefault;
-              background: @primaryColor;
+            .appHelp {
+              position: relative;
+              left: 5%;
+              bottom: 38%;
+              margin: 0;
             }
-            &.favorite:hover {
-              color: #797979;
-              background: transparent;
+          }
+          .applicationActions {
+            height: 22px;
+            justify-content: space-between;
+            padding: 0 0 0 8px;
+            a {
+              font-weight: 600;
             }
-            &.favorite:hover:disabled {
-              color: @baseBackgroundDefault;
-              background: @primaryColor;
+            button {
+              padding: 0;
+              .v-icon {
+                color: @darkYellowDefault!important;
+              }
             }
           }
         }
       }
-    }
-    .noApp {
-      margin-right: auto;
-      margin-left: auto;
-    }
-    .appPaginator {
-      padding-top: 30px;
-    }
-    .appPaginator button {
-      background: @primaryColor;
-      border: 1px solid @primaryColor;
-      color: @baseBackgroundDefault;
-      padding: 5px 10px;
-      border-radius: 5px;
-      box-shadow: 0 0 3px #ff7c4061;
-    }
-    .appPaginator button:hover {
-      background: transparent;
-      color: @primaryColor;
-    }
-    .appSearch {
-      .v-input__prepend-inner {
-        margin-top: 6px!important;
-      }
-      .v-text-field {
-        padding-top: 0!important;
-        margin-top: 0!important;
+      .noApp {
+        margin-right: auto;
+        margin-left: auto;
       }
-      .v-text-field__slot {
-        height: 48px!important;
-        &:focus {
-          outline: none!important;
-        }
+      .appPaginator {
+        padding-top: 30px;
       }
-      .v-input__slot {
-        height: 36px!important;
+      .appPaginator button {
+        background: @primaryColor;
+        border: 1px solid @primaryColor;
+        color: @baseBackgroundDefault;
+        padding: 5px 10px;
+        border-radius: 5px;
+        box-shadow: 0 0 3px #ff7c4061;
```

Comment:
Is there a variable here to use ?

Reply:
Couldn't find the variable, will verify for the right one.